### DISCUSSION TITLE
Does Excalidraw have a pencil tool?

### DISCUSSION THREAD

**Discussion by hazeymunster** [zsviczian/obsidian-excalidraw-plugin#2148] [Q&A]
Hello, I’m new to Obsidian and Excalidraw, so the answer to my question may be obvious, but I can’t find it.

While I have several apps for drawing, I really enjoy writing and drawing in my notes app, which is now Obsidian; I understand Obsidian is not for handwriting or drawing, but the Excalidraw plugin changes this as much as reasonably possible.

**However, is there a pencil tool in Excalidraw? Or is it only a pen tool?**

When I refer to a Pencil tool, I mean a _pencil-like drawing tool_ that looks like and has the quality of a pencil; I’m not referring to the Apple Pencil or actual stylus being used. I want to sketch, as well as write, in Pencil, in Obsidian. I like taking notes and sketching on the same page. Yes, other apps like Concept, GoodNotes, etc. would work better for this, but I really like Obsidian overall and having a pencil/sketching capability would be incredible. For example, something like the Apple Notes app’s pencil tool would be perfect (it has the best simple pencil tool, but Apple Notes’ level of data loss is bar none, so I do not use it anymore).

**Comment by LeonelRFF** [comment]
In the settings there is a section to add up to 10 brushes, this is added to the sidebar, by double clicking there you can edit it to have the order you want, thickness, color etc. 

![Screenshot_2024-12-11-22-18-40-77_51606159b24eff83e24a54116878fe3e](https://github.com/user-attachments/assets/e75f16dc-ad6e-48d7-b66c-5208d8c67a68)
![Screenshot_2024-12-11-22-22-26-68_51606159b24eff83e24a54116878fe3e](https://github.com/user-attachments/assets/fc70d64e-cae4-4721-8253-b39b8b3ec9db)
![Screenshot_2024-12-11-22-15-07-03_51606159b24eff83e24a54116878fe3e](https://github.com/user-attachments/assets/0bc955df-302c-4c9c-a84a-c8e8a1f16d70)


I am not a professional drawer but the type of pencil that can be useful for this can be the fountain pen here is an image of what I was able to do

![temp6405104736808457825](https://github.com/user-attachments/assets/96305ecc-bd63-4b74-a2dd-1f20da8c01a8)

By the way, that's not me, it just came to my mind. 🙂
